src/page/sinfo.rs

```diff
@@ -2,6 +2,8 @@
 
 use super::{Class, Flags};
 
+use core::arch::asm;
```

Comment:
You need to bump the MSRV to avoid the compiler errors. We're currently running tests with Rust 1.57. You need to bump this to 1.59 (when `asm` was stabilized).